opsdroid/web.py

```diff
@@ -113,12 +145,28 @@ def setup_webhooks(self, skills):
                     )
 
     async def web_index_handler(self, request):
-        """Handle root web request."""
+        """Handle root web request to opsdroid API.
+
+        Args:
+            request: web request to the root (index)
+
+        Returns:
+            dict: returns successful status code and greeting for the root page
+
+        """
         return self.build_response(200, {
             "message": "Welcome to the opsdroid API"})
 
     async def web_stats_handler(self, request):
-        """Handle stats request."""
+        """Handle stats request.
+
+        Args:
+            request: web request to render opsdroid stats
+
+        Returns:
+            dict: returns successful status code and dictionary with stats requested
```

Comment:
Also looks like this line is too long. Could you add a linebreak to make it less that 80 characters.